### ISSUE TITLE
Zero Clearance Around SPI and I2C connector on PolyVent SPI Controller Faceplate, Version8

### ISSUE THREAD

**Issue by ForrestErickson** [PubInv/general-purpose-alarm-device#233]
Fabricated out of PLA, this faceplate had zero clearance around the connectors.
![image](https://github.com/PubInv/general-alarm-device/assets/5836181/6506fd14-3bf0-4522-913a-bf1f1f81c30b)

File:
https://github.com/PubInv/general-alarm-device/blob/main/Hardware/PolyVent_SPI_Controller/CAD_model/PolyVent_SPI_Controller-faceplate%20v8.stl

**Comment by ForrestErickson** [comment]
Enlarged connector holes on the FreeCad 3D model.
![image](https://github.com/PubInv/general-alarm-device/assets/5836181/7cae0308-48b6-4a54-89f7-590a655c7795)

**Comment by ForrestErickson** [comment]
Updated file for larger connector holes. Fabricated on Prusa i3 MK3 using silver colored PLA.

![image](https://github.com/PubInv/general-alarm-device/assets/5836181/b4b4a666-223c-40a2-91bc-2d085daba378)
Note that the screw hole is not round. The 3mm screw will not go through this hole. I had to ream with a 1/8" drill for the 3mm screw to go.
The 3mm nut fit into the pocket with no interference and in fact fell out during assembly. 

The enlarged connector holes work.  I even made a cable with JST connector to test fit.
![image](https://github.com/PubInv/general-alarm-device/assets/5836181/a60b9908-22fd-4d7f-bdb8-212a08c428db)
